test/integration/targets/collections_internal_redirect/collections/ansible_collections/ns/coll/plugins/action/test_redirect.py

```diff
@@ -0,0 +1,60 @@
+# Copyright: (c) 2021, Ansible Project
```

Comment:
this has a lot of overlap with https://github.com/ansible/ansible/blob/devel/test/integration/targets/collections/collection_root_user/ansible_collections/testns/testcoll/plugins/action/plugin_lookup.py - even if you want to keep the dedicated action, this whole thing probably doesn't need to be a dedicated test target. Just add the custom bits you need under `collection_root_user` (as that's kind of the main one for collections). I actually considered making `plugin_lookup` a supported core action at one point- it might still be worth thinking about...

Reply:
Thanks, I hadn't noticed that action plugin! The diff is much smaller now.